microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -894,6 +905,48 @@ def verify_boot_error_fail_warnings(self, node: Node) -> None:
             f" {node.os.name} {node.os.information.version}"
         ).is_empty()
 
+    @TestCaseMetadata(
+        description="""
+        This test will check ERROR, WARNING messages from /var/log/cloud-init.log
+        and also check cloud-init exit status.
+
+        Steps:
+        1. Get ERROR, WARNING messages from /var/log/cloud-init.log.
+        2. If any unexpected ERROR, WARNING messages or non-zero cloud-init status
+         fail the case.
+        """,
+        priority=1,
+        requirement=simple_requirement(supported_platform_type=[AZURE, READY]),
+    )
+    def verify_cloud_init_error_status(self, node: Node) -> None:
+        cat = node.tools[Cat]
+        if isinstance(node.os, CBLMariner):
+            if node.shell.exists(node.get_pure_path("/var/log/cloud-init.log")):
+                log_output = cat.read("/var/log/syslog", force_run=True, sudo=True)
```

Comment:
I hit `cat: /var/log/syslog: No such file or directory` on MicrosoftCBLMariner azure-linux azure-linux-3 3.20240401.01

Reply:
the log should be /var/log/cloud-init.log